rhc-worker-playbook.spec.rpkg

```diff
@@ -28,17 +30,18 @@ export GRPC_PYTHON_BUILD_SYSTEM_OPENSSL=True
 export GRPC_PYTHON_BUILD_SYSTEM_ZLIB=True
 export GRPC_PYTHON_BUILD_SYSTEM_CARES=True
 export GRPC_PYTHON_DISABLE_LIBC_COMPATIBILITY=True
-%{__make} PREFIX=%{_prefix} LIBDIR=%{_libdir} installed-lib-dir
+%{__make} PREFIX=%{_prefix} LIBDIR=%{_libdir} installed-lib-dir SYSCONFDIR=%{_sysconfdir}
```

Comment:
Now that `rhc` is a package, I would suggest including it in `BuildRequires`, and then calling `pkg-config` to get the location of the worker config directory. It may or may not be under `%{_sysconfdir}`.

```
# Makefile
CONFIG_DIR=$(pkg-config rhc --variable workerconfdir`)
```

This could also be left as is; it will work. But as a future enhancement, it might be a good idea. So far, `rhc` does nothing special with that directory except define it in the pkg-config module, but things may change in the future.

Reply:
Now using pkg-config